express-server/index.js

```diff
@@ -0,0 +1,66 @@
+import express from "express";
+import cors from "cors";
+import bodyParser from "body-parser";
+import { TodoIMDBManager } from './DBManager/TodoIMDBManager.js';
+import { handleServerError } from './errors/utils.js';
+import { logger } from './logger/logger.js';
+import { getParentFolder } from './utils/folderUtils.js';
+import { eClientLocations } from './constants/clientLocations.js';
+
+const parentFolder = getParentFolder();
+const todoIMDBManager = new TodoIMDBManager();
+const app = express();
+const port = process.env.PORT || 8000;
+
+app.options("*", cors());
```

Comment:
cors is dangerous and not needed in this use case. let's discuss about how not to use it.